### PR TITLE
The next web branch

### PR DESCRIPTION
**Pull request by Alaska01**
-This project consists of replicating the The Next Web website putting emphasis on how it behaves differently depending on the size of the screen where the website is rendered.
-You can find the original project specification at: https://www.theodinproject.com/courses/html5-and-css3/lessons/building-with-responsive-design, but bear in mind that the design pattern for this site changes frequently, hence every detail upon submit might not look similar to the current one. The pictures are the major aspects that changes. Our site was responsive on all screens.

### CODE REVIEW

**Review by trekab — CHANGES_REQUESTED** [review]
# Changes Required! :tada: 
Hi @Alaska01 and @Godswilly! Well done with this project so far. There are some changes to need to consider in order to complete this project. These include;
- [ ] Your clone is broken for screens larger than 1440px. [Screenshot](https://github.com/trekab/images/blob/master/img/Aye/next_web.png?raw=true)
- [ ] On medium screens, your main navigation row has unnecessary white space(margin) on the rightmost end and left most end.
- [ ] The top 3 articles section:
  - On large and medium screens, the two smaller images on the rightmost part of the row (one above another) should have a smaller font compared to the bigger image on the left-most part.
  - On medium screens, the TWN logo should disappear ([link to page](https://thenextweb.com/)).
  - For large and medium screens, your text is not aligned appropriately. It should be aligned to the bottom left corners of the images.
- [ ] Your `Latest news` section is different from the original page on small screen sizes.
[Your clone](https://github.com/trekab/images/blob/master/img/Aye/latest_news_clone.png?raw=true) Vs [Original page](https://github.com/trekab/images/blob/master/img/Aye/latest_news.png?raw=true)
- [ ] The footer should not change through the break-points.
- [ ] Your CSS styles should use [media queries](https://www.w3schools.com/css/css_rwd_mediaqueries.asp) for 2 breakpoints ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/responsive_design/breakpoints_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59540217/)). 
  - No small breakpoint as it is mobile-first and the base styles should be for the mobile screen. ([Screenshot on mobile-first approach](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/responsive_design/mobile_first.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59540217/)) 
  - medium: min-width around 768px
  -  large: min-width 1024px

__Kindly make the necessary changes and submit another code review request.__
_Feel free to reach me on slack for any inquiries about this review._ :wink: 
__Treasure Kabareebe__
[GitHub](https://github.com/trekab) | [LinkedIn](https://www.linkedin.com/in/treasure-kabareebe/) | [Twitter](https://twitter.com/TKabareebe)

**Comment by Alaska01** [comment]
Made all the required changes  as requested by TSE. Kindly remember this site is progressive as updates are posted frequently by the developers.

**Review by BrittanyBlake — CHANGES_REQUESTED** [review]
### Status: Changes Required ♻️ 
****
### Hi @Alaska01  and @Godswilly ! 👋 
You have done a great job implementing the changes requested by the previous TSE! 👍 You have also done a wonderful job with your project so far! 💪 👏  There are just a few issues that need to be resolved before you can move on: ⬇️ 
****
- [ ] This project should not contain any media queries for the small screen as it is a mobile-first design. This was pointed out to you by the previous TSE. Please fix this.
Screen Shot 2020-06-17 at 2 00 18 AM

- [ ] There is a horizontal scroll on your page on the medium and small screen sizes. Please fix this.
Screen Shot 2020-06-16 at 11 32 07 PM





- **The top 3 articles section:**

- [ ] The text on the image in the bottom right of this section is overlapping on the medium screen. Please compare your clone with the original page and fix this. 
Your clone 👇 
Screen Shot 2020-06-16 at 11 33 27 PM
The image that I am referring to 👇 
Screen Shot 2020-06-16 at 11 33 41 PM
The original page 👇 
Screen Shot 2020-06-16 at 11 33 46 PM

- [ ] The text on the first image in this section on the small screen is also overlapping. 
      - It may be hard to see here but "Tech" is overlapping with "Divider"


Your clone 👇 
Screen Shot 2020-06-17 at 1 50 43 AM
The original page 👇 
Screen Shot 2020-06-17 at 1 51 49 AM

- **Footer**

- [ ] As was pointed out to you by the previous TSE, the footer should not change with the breakpoints. In your project, the footer is not properly aligned on the medium screen. Please compare your clone with the original and fix this. 
Project requirements 👇 
Screen Shot 2020-06-17 at 1 56 05 AM
Your clone 👇 
Screen Shot 2020-06-17 at 1 38 26 AM
The original page 👇 
Screen Shot 2020-06-17 at 1 38 49 AM








****
### Please make the required changes and resubmit for a new code review once you are ready 🙏

Good Luck and Happy Coding 🍀 👩🏾‍💻

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Brittany Blake)._

**Comment by Alaska01** [comment]
Many thanks for the feedback on the reviews. We have done it the way you asked. Well, we are learning and are always very happy to get more knowledge. 

All changes have been made as required. We would be ever willing to look at some suggestions that you may have. Kindly consider the points below as well.

Kindly use the Mozilla Firefox developers tools instead of Google Chrome to see that the site is not overflowing on mobile, that problem is specific to the Chrome developers tool. We currently have do not have control over that; it's a bug at their end.

It would be great if you can share with us where it is a fixed rule that mobile-first should not have a media query. We would be thrilled to update our knowledge. But looking at W3School and Free Code Camp, it seems apparent that one can set as many breaking points for design. It is even best practice to have numerous breaking point beyond just two. Here are some resources from leading industry professionals we were following (https://www.w3schools.com/css/css_rwd_mediaqueries.asp, https://www.freecodecamp.org/news/the-100-correct-way-to-do-css-breakpoints-88d6a5ba1862/).

We may be wrong with our knowledge of grammar, but the last time we checked a "HINT" does not translate into a fixed rule that must be followed. As we are always saying, we are newbies in the programming world, feel free to educate us on anything we don't know but feel we should be conversant in such areas. We are always open to suggestions and learning. Thank you very kindly.

**Comment by addod19** [comment]
## Pull Request under review by Daniel Larbi Addo (TSE)

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes :recycle: :traffic_light: 

Hi **TEAM**, great job on fixing some of the comments made by the previous TSE :+1: However, I would like to highlight some of the comments made earlier so we can fix it and get our project approved. Kindly find comments below:
- ``` Mobile-first design ```, In mobile-first design, we do not need a media query. Let us remove the media query and fix the issue as pointed out earlier
- ``` Horizontal scroll ```, as developers, we create an application to fit on all browsers. So a quick fix for the horizontal scroll would be to set the main container width to 100% :slightly_smiling_face: 

Kindly make these changes and submit another code review :smile: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Comment by Alaska01** [comment]
Fixed the issues as requested by the TSE. Thanks for the feedback it was useful. If at all the google developers tool are misbehaving try opening it on real devices, the live link works fine. I hope this works perfect at your end too. Thanks.

**Review by tirthajyoti-ghosh — CHANGES_REQUESTED** [review]
# Status: Changes required 🚨

## Nice job making the changes 👌🙌👏

### But we need to make some more changes.

- I see that you have forgotten to exclude the `node-modules` folder when committing. Remember to add that folder in your `.gitignore` file. Also, remove the folder from your repo.

- Let's make sure that we have the TNW logo in large screens

### Clone

image

### Original

image

- The title must come down. Right now it has extra space below it. Also, you are missing the category over the title.

### Clone

Annotation 2020-06-19 002549

### Original

Annotation 2020-06-19 002629

- Let's also add some space below each section -> `top 3 articles`, `latest news`, `latest funding`, `apps, gear, tech, ...` and `latest deals`.

- Elements here are squeezed together. Add some space around them.

### Clone

Annotation 2020-06-19 004756

### Original

Annotation 2020-06-19 004827

- The list items are squeezed together in `apps, gear, tech,...` section. Let's add some padding to them.

### Clone

Annotation 2020-06-19 003503

### Original

Annotation 2020-06-19 003529

- The image and the title in `latest deals` are not aligned properly.

### Clone

Annotation 2020-06-19 004249

### Original

Annotation 2020-06-19 004317

- There are couple of invalid CSS properties. I have pointed them out in the comments below.

Overall, you have done a great job with this project. Apply the suggested changes and request another review.

Happy coding 👨‍💻

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,2043 @@
...
+.main-grid .box1 div {
+  position: absolute;
+  top: 0;
+  width: 95%;
+  height: 200px;
+  font-size: 1.2rem;
+  line-height: 1.5;
+  padding-left: 1.5%;
+  box-sizing: border-box;
+  background-color: red 5px solid;
```
Comment (tirthajyoti-ghosh) [inline css/style.css]: This is an invalid property. `background-color` only takes color values. If you want to add other background properties, use the shorthand `background` property.

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,2043 @@
...
+}
+
+.ft-1b {
+  display: flex;
+  justify-content: center;
+  height: 50px;
+  margin: 0 60px;
+  color: white;
+  font-size: 11px;
+  margin-top: auto auto;
```
Comment (tirthajyoti-ghosh) [inline css/style.css]: Invalid property. `margin-top` only takes one value.

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,2043 @@
...
+    background:
+      linear-gradient(
+        rgba(255, 31, 42, 0.7) 0%,
+        rgba(0, 0, 0, 0.9) 100%
+      ),
+      url("images/trump-tweets.png") no-repeat;
+  }
+
+  .containers-latest-deals div {
+    margin-top: 10px 10px;
```
Comment (tirthajyoti-ghosh) [inline css/style.css]: Invalid property. `margin-top` only takes one value.

File: css/style.css [hunk 4]
```diff
@@ -0,0 +1,2043 @@
...
```
Comment (tirthajyoti-ghosh) [inline css/style.css]: Invalid property. `margin-top` only takes one value.

**Review by tirthajyoti-ghosh — COMMENTED** [review]
Apply these changes and request another code review.

Happy coding 👨‍💻

**Review by HADDADSOHAIB — CHANGES_REQUESTED** [review]
### Status: Change Required :recycle: 

### Hello @Alaska01     :hand: 

### Good job your project is getting better and better. :+1: 

### Unfortunately, there are still one change that must be done to get the project approved:

Screenshot from 2020-06-19 14-00-51
You add the node module to gitignore, but you need also to remove it from the repository.
- You have commented code in you HTML, remove that.
### Once you are done, ask for another code review

**Comment by Alaska01** [comment]
Removed HTML comments and node modules from repo

**Review by christianotieno — APPROVED** [review]
# PROJECT APPROVED :champagne:
Good job working on the changes :fireworks: 
##
***All the required tasks have been fulfilled in this project. I will go ahead and mark this as complete.***
- Congratulations on the splendid job :+1:
- You can now merge this. I wish you all the very best in your next tasks 🥇 
##
Happy coding :smile:
**[Christian Otieno](https://github.com/ChristianOtieno)**
[Twitter](https://twitter.com/iamchrisotieno) [linkedin](https://www.linkedin.com/in/christianotieno/) [GitLab](https://gitlab.com/christianotieno)

**Pull request merged**